cpp/flightsql/gizmosql/README.md

```diff
@@ -0,0 +1,105 @@
+<!--
+Copyright 2025 Columnar Technologies Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Connecting C++ and GizmoSQL (an Arrow Flight SQL server - powered by DuckDB or SQLite) with ADBC
+
+## Instructions
+
+This example uses [GizmoSQL](https://gizmodata.com/gizmosql), but other open source tools and vendor products that support Arrow Flight SQL will also work with this driver.
+
+> [!TIP]
+> If you already have a GizmoSQL instance running, skip the steps to set up GizmoSQL.
+
+### Prerequisites
+
+1. [Install uv](https://docs.astral.sh/uv/getting-started/installation/)
+
+1. [Install miniforge](https://github.com/conda-forge/miniforge)
+
+1. Create and activate a new environment with the required C++ libraries:
+
+   ```sh
+   mamba create -n adbc-cpp -c conda-forge cmake compilers libadbc-driver-manager libarrow
+
+   # Initialize mamba in your shell if not already done
+   eval "$(mamba shell hook --shell zsh)"
+   mamba activate adbc-cpp
+   ```
+
+   (`cmake` is only needed if you use CMake to build the C++ program below.)
+
+### Set up GizmoSQL server (if you don't already have one)
+
+1. [Install podman](https://podman.io/docs/installation)
+
+1. Start the GizmoSQL server:
+```bash
+podman run --name gizmosql \
+           --detach \
+           --rm \
+           --tty \
+           --init \
+           --publish 31337:31337 \
+           --env TLS_ENABLED="1" \
+           --env GIZMOSQL_PASSWORD="gizmosql_password" \
+           --env PRINT_QUERIES="1" \
+           --pull always \
+           gizmodata/gizmosql:latest
+```
```

Comment:
Could you use docker instead of podman as like the trino example?
Something like:

```sh
docker run -d --name gizmosql -p 31337:31337 gizmodata/gizmosql:latest-slim
```

Reply:
is it essential that we switch to Docker?  Podman is a good open-source alternative to Docker.  I can do it - but just wanted to understand why...

Reply:
> is it essential that we switch to Docker? Podman is a good open-source alternative to Docker. I can do it - but just wanted to understand why...

People who use podman can probably translate docker commands into podman commands, but I don't think everyone who knows Docker can translate podman commands into docker commands.
I think it doesn't really matter what users use to run the container here, so we should use something more understandable IMO.

> Is it ok to keep the multi-line format?

I think it should be avoided because the difference in newline characters will prevent it from running on PowerShell.